server/Arcadia.Assistant.Organization/EmployeeSearch.cs

```diff
@@ -36,15 +36,24 @@ protected override void OnReceive(object message)
             switch (message)
             {
                 case StartSearch:
-                    foreach (var actorRef in this.actorsToReply)
+
+                    foreach (var actorRef in this.employeeActorsToReply)
                     {
                         actorRef.Tell(EmployeeActor.GetEmployeeInfo.Instance);
                     }
 
-                    if ((this.requesters.Count == 0) || (this.actorsToReply.Count == 0))
+                    if ((this.requesters.Count == 0) || (this.employeeActorsToReply.Count == 0))
```

Comment:
Any() ?

Reply:
no. Any and Count have different semantics.